packages/wagmi/tsconfig.json

```diff
@@ -6,5 +6,5 @@
     "baseUrl": "../../"
   },
   "extends": "../../tsconfig.json",
-  "include": ["exports", "src"]
+  "include": ["exports", "src", "src/**/*.json"]
```

Comment:
Same, is this intentional? We already have `src` file listed in `include` array.